velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -208,5 +214,42 @@ TEST_F(DateTimeFunctionsTest, lastDay) {
   EXPECT_EQ(lastDayFunc(std::nullopt), std::nullopt);
 }
 
+TEST_F(DateTimeFunctionsTest, dateSub) {
+  const auto dateSubInt32 = [&](std::optional<Date> date,
+                                std::optional<int32_t> value) {
+    return evaluateOnce<Date>("date_sub(c0, c1)", date, value);
+  };
+  const auto dateSubInt16 = [&](std::optional<Date> date,
+                                std::optional<int16_t> value) {
+    return evaluateOnce<Date>("date_sub(c0, c1)", date, value);
+  };
+  const auto dateSubInt8 = [&](std::optional<Date> date,
+                               std::optional<int8_t> value) {
+    return evaluateOnce<Date>("date_sub(c0, c1)", date, value);
+  };
+
+  // Check null behaviors
```

Comment:
Remove these tests. They test the framework, not the function.

Reply:
@mbasmanova Checked the Spark and Presto tests in DateTimeFunctionsTest.cpp, looks all used like this style. Can you check further. Thanks.

Reply:
These test may need to be updated as well. Reviewers do not always catch everything.

Reply:
Done